addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -93,6 +93,20 @@ def test_message_parse_xhtml(self):
         # Test that the parsing of XHTML mails does not fail
         self.env['mail.thread'].message_parse(self.from_string(test_mail_data.MAIL_XHTML))
 
+    def test_message_parse_and_replace_binary_octetstream(self):
```

Comment:
As tests are executed by alphabetical order, could you move it at the right place in the class :D ? That way when you run tests, it follows the same order. Not a crucial comment but just for sanity :)